src/main/java/bowling/model/state/Start.java

```diff
@@ -0,0 +1,24 @@
+package bowling.model.state;
+
+import bowling.model.Score;
+import bowling.model.state.finishedState.Strike;
+
+public class Start extends State {
+    private final static String EXPRESSION = " ";
+
+    @Override
+    public State bowling(int fallenPin) {
+        score = Score.from(fallenPin);
+
+        if(score.isMaxScore()){
```

Comment:
조건문 사용시 다음과 같이 if 와 괄호 사이에 공백을 넣어주시는게 컨벤션이오니 참고 부탁드려요 :)
[7.4절](https://myeonguni.tistory.com/1596)
https://naver.github.io/hackday-conventions-java/#space-between-keyword-parentheses

```suggestion
        if (score.isMaxScore()) {
```